db/data_migration/20181001113221_retag_hca_to_rsa.rb

```diff
@@ -0,0 +1,358 @@
+  'regulatory-judgement-waterloo-housing-group-limited',
+  'regulatory-judgement-watford-community-housing-trust',
+  'regulatory-judgement-watmos-community-homes',
+  'regulatory-judgement-weaver-vale-housing-trust-limited',
+  'regulatory-judgement-wellingborough-homes-limited',
+  'regulatory-judgement-west-kent-housing-association',
+  'regulatory-judgement-westward-housing-group-limited--2',
+  'regulatory-judgement-wirral-partnership-homes-limited',
+  'regulatory-judgement-wm-housing-group-limited',
+  'regulatory-judgement-worthing-homes-limited',
+  'regulatory-judgement-wrekin-housing-group-limited',
+  'regulatory-judgement-wythenshawe-community-housing-group-limited',
+  'regulatory-judgement-yarlington-housing-group',
+  'regulatory-judgement-yorkshire-coast-homes-limited',
+  'regulatory-judgement-yorkshire-housing-limited--2',
+  'regulatory-judgement-your-housing-group-limited--2',
+  'regulatory-judgements-and-regulatory-notices',
+  'rent-standard-guidance',
+  'restructures-and-constitutional-changes',
+  'rsh-stakeholder-survey',
+  'sector-risk-profile-2013',
+  'sector-risk-profile-2014',
+  'sector-risk-profile-2015',
+  'sector-risk-profile-2016',
+  'sector-risk-profile-2017',
+  'service-standard-performance-data',
+  'statistical-data-return-survey-responses',
+  'the-accounting-direction-for-social-housing-in-england-from-april-2012',
+  'the-accounting-direction-for-social-housing-in-england-from-april-2015',
+  'value-for-money-code-of-practice',
+  'value-for-money-metrics-technical-note',
+  'what-is-the-regulatory-framework',
+  'current-registered-providers-of-social-housing',
+  'memorandum-of-understanding-between-the-regulator-of-social-housing-and-the-housing-ombudsman',
+  'innovation-plan-hca-regulator-of-social-housing',
+  'regulatory-judgement-trinity-housing-association-limited',
+  'value-for-money-summary-and-techical-reports',
+  'decision-statement-on-introducing-fees-for-social-housing-regulation',
+  'decision-statement-on-an-amendment-to-the-tenant-involvement-and-empowerment-standard',
+  'decision-statement-on-registration-criteria',
+  'decision-statement-on-changes-to-the-accounting-direction',
+  'decision-statement-on-the-value-for-money-standard',
+  'decision-statement-on-changes-to-regulatory-framework',
+  'leasehold-schemes-for-the-elderly-management-fee-limit',
+  'regulator-of-social-housing-business-impact-target',
+  # statistics
+  'statistical-data-return-2011-to-2012',
+  'statistical-data-return-2012-to-2013',
+  'statistical-data-return-2013-to-2014',
+  'statistical-data-return-2014-to-2015',
+  'statistical-data-return-2015-to-2016',
+  'statistical-data-return-2016-to-2017',
+].each do |slug|
+  begin
+    document = Document.find_by(slug: slug)
+    edition = document.latest_edition
+
+    lead_organisations = edition.lead_organisations.to_a
+    next if lead_organisations.include? regulator_social_housing
+    lead_organisations << regulator_social_housing
```

Comment:
Do we need to clear which lead organisation the document was previously tagged to? I notice the Zendesk ticket says "retag"?

Reply:
It does, but the latest comment says

> Just to confirm that https://www.gov.uk/government/organisations/regulator-of-social-housing (https://www.gov.uk/government/organisations/regulator-of-social-housing) is now live and most things seem right.
>
> Once the publications are tagged on it (alongside HCA) that’d be great.

Reply:
Ah missed that, okay!